plugins/modules/cloudwatchevent_rule.py

```diff
@@ -84,15 +84,29 @@
         type: str
         description:
           - A JSON object that will override the event data when passed to the target.
-          - If neither I(input) nor I(input_path) is specified, then the entire
-            event is passed to the target in JSON form.
+          - If neither I(input) nor I(input_path) nor I(input_paths_map) nor I(input_template)
+            is specified, then the entire event is passed to the target in JSON form.
       input_path:
         type: str
         description:
           - A JSONPath string (e.g. C($.detail)) that specifies the part of the event data to be
             passed to the target.
-          - If neither I(input) nor I(input_path) is specified, then the entire
-            event is passed to the target in JSON form.
+          - If neither I(input) nor I(input_path) nor I(input_paths_map) nor I(input_template)
+            is specified, then the entire event is passed to the target in JSON form.
+      input_paths_map:
```

Comment:
`version_added: 1.6.0` is missing for the submodules. But maybe next release will be `2.0.0`. I've no idea

Reply:
2.0.0 will be the next one

Reply:
As @markuman suggested, could you please add `version_added: 2.0.0` for each option?

Reply:
I added it, however I didn't find any similar pattern in some random modules I browsed to get some reference on how to do it, so let me know if it's ok.